docs/man_pages/general/autocomplete-status.md

```diff
@@ -1,22 +1,20 @@
 autocomplete status
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder autocomplete status`
 
-
-Shows if autocompletion is enabled for bash and zsh.
+Prints your current command-line completion settings.
```

Comment:
Is it good idea to explicitly say that this command won't work on Windows with CMD as a shell? The sub-commands already say it.